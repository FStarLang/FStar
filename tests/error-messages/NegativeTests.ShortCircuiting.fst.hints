[ "Ìuz±?:×Q…\u001e\u0010 ‚øÝJ", [] ]