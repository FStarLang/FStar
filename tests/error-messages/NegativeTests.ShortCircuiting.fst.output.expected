* Warning 240 at NegativeTests.ShortCircuiting.fst(23,4-23,6):
  - NegativeTests.ShortCircuiting.ff is declared but no definition was found
  - Add an 'assume' if this is intentional

* Warning 240 at NegativeTests.ShortCircuiting.fst(19,4-19,7):
  - NegativeTests.ShortCircuiting.bad is declared but no definition was found
  - Add an 'assume' if this is intentional

>> Got issues: [
* Error 19 at NegativeTests.ShortCircuiting.fst(21,16-21,33):
  - Subtyping check failed
  - Expected type b: Prims.bool{bad_p b} got type Prims.bool
  - The SMT solver could not prove the query. Use --query_stats for more
    details.
  - See also NegativeTests.ShortCircuiting.fst(19,31-19,38)

>>]
>> Got issues: [
* Error 19 at NegativeTests.ShortCircuiting.fst(25,11-25,36):
  - Assertion failed
  - The SMT solver could not prove the query. Use --query_stats for more
    details.
  - See also Prims.fst(459,90-459,102)

>>]
* Warning 240 at NegativeTests.ShortCircuiting.fst(25,0-25,36):
  - Missing definitions in module NegativeTests.ShortCircuiting:
      bad
      ff

